source/benchmarking/benchmarking.md

```diff
@@ -410,36 +416,144 @@ Dataset size: For score purposes, the dataset size for a task is the size of the
+| After task  | n/a                                                                                                                                                                                                                                                                                                    |
+| Teardown    | Drop the `perftest` database.                                                                                                                                                                                                                                                                          |
+
+#### Large doc Client BulkWrite insert
+
+Summary: This benchmark tests driver performance inserting multiple large documents to the database using the `Client::bulkWrite` operation.
+
+Dataset: The dataset, designated LARGE_DOC consists of a JSON document with an encoded length of approximately 2,500,000
+bytes.
+
+Dataset size: For score purposes, the dataset size for a task is the size of the single-document source file (2,731,089
+bytes) times 10 operations, which equals 27,310,890 bytes or 27.31 MB.
+
+| Phase       | Description                                                                                                                                                                                                                                                                                        |
+|-------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Setup       | Construct a MongoClient object. Drop the `perftest` database. Load the LARGE_DOC dataset into memory as a language-appropriate document type (or JSON string for C). Create a list of insert models for 10 copies of the document, allowing the driver or database to auto-assign the `_id` field. |
+| Before task | Drop the `corpus` collection. Create an empty `corpus` collection with the `create` command. Construct a Collection object for the `corpus` collection to use for insertion.<br/>                                                                                                                  |
+| Do task     | Do an ordered `Client::bulkWrite` in the same namespace ("perftest.corpus") with the list of insert models.                                                                                                                                                                                        |
```

Comment:
I think the namespace is part of the model construction/setup phase.

Reply:
Since the model construction is very much linked with namespace selection, I thought implementers will understand that if even I mention the namespace requirements in the Do Task, it will be done in the setup phase. But I'll move the namespace requirement to the setup phase to be more direct.